source/app.d

```diff
@@ -19,12 +19,14 @@ import session.framesend;
 import session.tracking.expr;
 import std.process;
 
+version(linux) {
+    import dportals.filechooser;
+    import dportals.promise;
+}
 
 void main(string[] args) {
     insLogInfo("Inochi Session %s, args=%s", INS_VERSION, args[1..$]);
     
-    // Force X11 as the video driver on Linux
```

Comment:
This should probably be seperate from this PR

Reply:
oh, yeah, it should. mb.